Source/WebCore/platform/audio/FFTConvolver.cpp

```diff
@@ -64,28 +64,30 @@ void FFTConvolver::process(FFTFrame* fftKernel, const float* sourceP, float* des
     size_t numberOfDivisions = halfSize <= framesToProcess ? (framesToProcess / halfSize) : 1;
     size_t divisionSize = numberOfDivisions == 1 ? framesToProcess : halfSize;
 
-    for (size_t i = 0; i < numberOfDivisions; ++i, sourceP += divisionSize, destP += divisionSize) {
+    for (size_t i = 0; i < numberOfDivisions; ++i, source = source.subspan(divisionSize), destination = destination.subspan(divisionSize)) {
         // Copy samples to input buffer (note contraint above!)
-        float* inputP = m_inputBuffer.data();
+        auto inputP = m_inputBuffer.span();
 
         // Sanity check
-        bool isCopyGood1 = sourceP && inputP && m_readWriteIndex + divisionSize <= m_inputBuffer.size();
+        bool isCopyGood1 = source.data() && inputP.data() && m_readWriteIndex + divisionSize <= m_inputBuffer.size();
         ASSERT(isCopyGood1);
         if (!isCopyGood1)
             return;
 
-        memcpy(inputP + m_readWriteIndex, sourceP, sizeof(float) * divisionSize);
+        IGNORE_WARNINGS_BEGIN("restrict")
```

Comment:
Very surprising that we both get a warning here and that we need to ignore it.

Reply:
Only on the WPE bot.